templates/main/about/partners.html

```diff
@@ -0,0 +1,48 @@
+{% extends "main/base.html" %}
+{% block title %}Partners{% endblock %}
+{% block og_title %}Partners{% endblock %}
+{% block og_description %}List and Links to Partner servers{% endblock %}
+{% block content %}
+    <div class="uk-section">
+        <div class="uk-container uk-container-small">
+            <article class="uk-article">
+                <h1 class="uk-article-title hover-title" id="top">
+                    Partners
+                    <a href="#top" class="uk-text-primary" title="Permanent link to this header">
+                        <i class="fas fa-paragraph" data-fa-transform="shrink-8"></i>
+                    </a>
+                </h1>
+                <div class="uk-column-1-2@m uk-column-divider">
+                    {% if partners is none %}
+                        <div class="uk-alert-danger" uk-alert>
+                            <p>
+                                We were unable to load the <code>partners.json</code> file. If you see this, please
+                                notify us!
+                            </p>
+                        </div>
+                    {% else %}
+                        {% for partner_name, partner_data in partners.items() %}
+                            <div class="uk-inline">
+                                <div class="uk-card uk-card-hover uk-card-default" id="partner-card">
```

Comment:
id's must be unique, so I'm afraid this isn't good enough. You've got the right idea, though. how about you put an ID on the `<div class="uk-column-1-2@m uk-column-divider">`, and use a selector that says "select the divs inside of a div with this id, but only if they have the class `uk-card`. That's very specific so it won't accidentally style someone elses code, and it also won't apply to the uk-alert-danger div.